scripts/set-dm-password-by-email.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
Can we call this script something else? It kind of implies we are setting a new password by sending an email 😸 

How about `set-user-password-via-apiclient.py`?

Reply:
oh... yeah I guess it does. But "via apiclient" is a bit obvious. How else could it? What if it were `set-user-password-by-email-address.py`?